src/testUtils/publicKeyGenerator.ts

```diff
@@ -0,0 +1,16 @@
+import { generateKeyPair } from 'node:crypto';
+import { promisify } from 'node:util';
+
+const generateKeyPairAsync = promisify(generateKeyPair);
+export const generatePublicKey = async () => {
```

Comment:
FYI, I ended up implementing something equivalent in #76 (`generateKeyPair()`), which I think we should use instead: it also outputs a private key, which we'll need in some cases.